packages/errors/src/react/error-page.tsx

```diff
@@ -67,39 +70,51 @@ const styles = {
     )
   }
 }
-
-const SearchForm = props => (
-  <form action={props.action} method="get" {...styles.search(props)}>
-    <div {...styles.searchIcon(props)}>{icons.search()}</div>
+interface SearchFormProps extends HTMLAttributes<HTMLFormElement> {
+  action?: string
+  themeName: ValueOf<typeof themeNames>
+}
+const SearchForm: React.FC<SearchFormProps> = ({
+  action,
+  themeName,
+  ...props
```

Comment:
I'm all for our other pattern of `rest`-named vars in order to help disambiguate from original/full `props`